receiver/splunkhecreceiver/receiver.go

```diff
@@ -245,7 +253,7 @@ func (r *splunkReceiver) handleRawReq(resp http.ResponseWriter, req *http.Reques
 	}
 
 	if req.ContentLength == 0 {
-		r.obsrecv.EndLogsOp(ctx, typeStr, 0, nil)
```

Comment:
I don't think the `EndLogsOp` should be removed? It should still be observed it ended.